README.md

```diff
@@ -87,22 +97,36 @@ Create the following configuration file in `workbench/settings_pb.py`:
 from settings import *
 
 INSTALLED_APPS += ('problem_builder',)
-DATABASES['default']['NAME'] = 'var/workbench.db'
 ```
 
-Because this XBlock uses a Django model, you need to sync the database
-before starting the workbench. Run this from the XBlock repository
-root:
+By default, the `xblock-sdk` uses the SQLite database but MySQL
+can be used by specifying an environment variable `WORKBENCH_DATABASES` in
+the following format.
+
+```bash
+export WORKBENCH_DATABASES='{"default": {"ENGINE": "django.db.backends.mysql", "NAME": "db", "USER": "root", "PASSWORD": "rootpw", "HOST": "127.0.0.1", "PORT": "3306"}}'
+```
```

Comment:
@rusrushal13, I think this paragraph, can be moved to after the `docker run` command. What do you say?

Reply:
First I thought the same, but running the export command before running the database looks better to me, that's why I have kept it before the docker command. What do you think?